eng/environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
```

Comment:
Any reason to not add this into the existing globals.yml file?

Reply:
Nope! Going to use it. Aside from this, the `environment.yml` is a conda specific thing.